### PR TITLE
Add retry for `deploy_compiled_contract`

### PR DESCRIPTION
**Pull request by enriquefynn**

### CODE REVIEW

**Review by mksh — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -61,6 +61,10 @@ echo "POSSIM_HOSTNAME=possim.devel" >> .env
 echo "COMPOSE_FILE=compose-host.yaml" > .env
 echo "POSSIM_HOSTNAME=127.0.0.1" >> .env
 ```
+* If running on OrbStack use the following hostname instead:
+```bash
+echo "POSSIM_HOSTNAME=possim.${COMPOSE_PROJECT_NAME}.orb.local" >> .env
```
Comment (QAston) [inline README.md]: does orbstack not apply the docker compose flag?

Comment (QAston) [reply README.md]: see compose-depod.yaml.

Also, if you want to merge this you should move this to the devpod section, right now it looks like you're talking native linux

Comment (mksh) [reply README.md]: I think orbstack doesn't support forwarding into Mac host interface (at least that's info i found on the inernet)

Comment (QAston) [reply README.md]: yeah, but in this case we're inside of a devpod so that shouldn't be a problem. The problem seems to be old docker compose impl?

Comment (QAston) [reply README.md]: @mksh turns out the issue was using docker-compose instead of docker compose

**Pull request merged**